src/site/content/en/notifications/codelab-notifications-service-worker/index.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: codelab
+title: Use a Service Worker to manage notifications
+authors:
+  - katejeffreys
+description: |
+  In this codelab, learn how to manage notifications with a
+  service worker.
+date: 2019-10-14
+# Add an updated date to your post if you edit in the future.
+# updated: 2019-06-27
+glitch: codelab-notifications-service-worker
+# Specify which file the glitch should start on.
+glitchPath: public/index.js  
+related_post: use-push-notifications-to-engage-users
+draft: true
+---
+
+In this codelab, you'll use a service worker
+to manage [notifications](/use-push-notifications-to-engage-users).
+The instructions here assume that you're already familiar with
+service workers and the basics of requesting notification permission and sending notifications.
+If you need a refresher on notifications, see the
+[Get started with the Notifications API](/codelab-notifications-get-started)
+codelab. To learn more about service workers, see Matt Gaunt's [Introduction to service workers](https://developers.google.com/web/fundamentals/primers/service-workers).
+
+## Remix the sample app and view it in a new tab
+
+Notifications are automatically blocked from the embedded Glitch app, so you won't be able to preview the app on this page. Instead, here's what to do:
+
+{% Instruction 'remix', 'ol' %}
+{% Instruction 'preview', 'ol' %}
+
+The Glitch should open in a new Chrome tab.
```

Comment:
```suggestion
The Glitch should open in a new browser tab.
```

Reply:
I left this as Chrome because later steps rely on Chrome DevTools.

Reply:
Yeah I realized that as I was reviewing it. I think in other places we just say "a new tab" don't we?

Reply:
Dunno, tbh. But my feeling is if people need to be in Chrome, they shouldn't discover that halfway through the process. Maybe we need another instruction widget variant that explicitly tells people to open in Chrome when a codelab is using DevTools.

Reply:
hm yeah, now that I'm poking around I see that we do a mix of things. Sometimes we tell folks that things will open a new tab, sometimes we directly mention Chrome DevTools.

Honestly having a note at the top that mentions that we're using Chrome DevTools in the codelab doesn't seem like a bad idea to have across the site. @kaycebasques wdyt?

Reply:
I went ahead and did it:

![image](https://user-images.githubusercontent.com/14199217/67965562-7d3d3780-fbbf-11e9-8bc8-9bce21831ee9.png)

If/when folks are happy with it, I can submit a separate PR to add it to the other codelabs that rely on DevTools.

Reply:
@kaycebasques and @katejeffreys, WDYT?

Reply:
@kaycebasques, friendly ping. If you're OK with this, I can apply it to other relevant codelabs.

Reply:
Hey, @kaycebasques and @robdodson, I just bumped into the need for this include again, so I'm gonna go ahead and do it in a separate PR since I don't know if/when this PR will be merged.

Reply:
Added a TODO in #2043 to mention the Chrome dependency one way or another